lib/ssz_types/beacon_chain/execution_payload_header.ex

```diff
@@ -50,4 +68,8 @@ defmodule SszTypes.ExecutionPayloadHeader do
   def decode(%__MODULE__{} = map) do
     Map.update!(map, :base_fee_per_gas, &Ssz.decode_u256/1)
   end
+
+  def default() do
```

Comment:
Lint fails because of this
```suggestion
  def default do
```